ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -642,6 +648,18 @@ def ocs_install_verification(
     health_check_tries = 20
     health_check_delay = 30
     if post_upgrade_verification:
+        # remove ceph crashes after upgrade due to bug https://bugzilla.redhat.com/show_bug.cgi?id=2233762
```

Comment:
I think we got the new bug: https://bugzilla.redhat.com/show_bug.cgi?id=2237861 , so probably both needs to be fixed?

Reply:
done